recipes/default.rb

```diff
@@ -20,8 +20,6 @@
 
 include_recipe 'varnish::repo' if node['varnish']['use_default_repo']
 
-package 'varnish'
-
```

Comment:
`template "#{node['varnish']['dir']}/#{node['varnish']['vcl_conf']}"` needs `directory "#{node['varnish']['dir']}/"` to be created before it.